src/Writers/LaTeXWriter.jl

```diff
@@ -574,7 +574,7 @@ end
 # equations in the Julia manual. However, if the equation is already wrapped in
 # align/align*, then there is no need to further wrap it (in fact, it will break).
 function latex(io::IO, math::Markdown.LaTeX)
-    if occursin(r"^\\begin\{align\*?\}", math.formula)
+    if occursin(r"^\\begin\{((equation)|(align)|(gather)|(flalign)|(multline)|(alignat)|(split))\*?\}", math.formula)
```

Comment:
I believe the brackets here are unnecessary:

```suggestion
    if occursin(r"^\\begin\{(equation|align|gather|flalign|multline|alignat|split)\*?\}", math.formula)
```